### PR TITLE
Time conversion done using array of timezones

### PR DESCRIPTION
**Pull request by BlackRoseRipp**
Added time conversion algorithm.
Functions off of JD's timezone array he created

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,78 +1,375 @@
...
-const client = new Discord.Client();
+const client = new Discord.Client()
 client.on('ready', () => {
-    console.log('The bot is connected to Yang\'s Gang!');
-});
+    console.log("The bot is connected to Yang's Gang!")
+})
 client.login(process.env.BOT_TOKEN)
 
+const timezoneCodes = [
```
Comment (ajones55555) [inline app.js]: Can you move this array declaration and also the currency codes array declaration to another file?
Then at the top of this file, do something like `const { currencyList, timezoneList } = require('./supportedConversions')`

File: app.js [hunk 2]
```diff
@@ -1,78 +1,375 @@
...
+    'KRAT',
+    'CST',
+    'JST',
+    'AEST',
+    'SAKT',
+    'NZST',
+]
+
+function timeConversion(msg) {
+    const content = msg.content.slice(10).toUpperCase().trim().split(' ') //10 is the length of '!DocBrown '
```
Comment (ajones55555) [inline app.js]: Hardcoding these numbers isn't really a sustainable approach as we keep adding more commands.
Can you add an enum object _near the top of the file_ to track supported commands?
For example: 
```
// at the top
const supportedCommands = {
     convertTimes: '!DocBrown',
     setup: '!Setup'
}
```

and then anywhere where we need !DocBrown or !Setup, replace it with `supportedCommands.________`

**Comment by JDPablo** [comment]
Getting hour times greater than 12 sometimes appended by am or pm (try EST->JST 11:45am)

**Comment by JDPablo** [comment]
Line 46 crashes bot attempting to run "!DocBrown EDT->JDT 11:45am"

**Review by ajones55555 — CHANGES_REQUESTED** [review]

File: app.js (hunk 2 above)
Comment (BlackRoseRipp) [reply app.js]: Added it. We could add that to the supportedConversions file instead if we keep adding commands

**Review by ajones55555 — APPROVED** [review]

**Pull request merged**